sql/messaging_system_derived/cfr_users_last_seen_v1/query.sql

```diff
@@ -0,0 +1,47 @@
+WITH _current AS (
+  SELECT
+-- In this raw table, we capture the history of activity over the past
+-- 28 days as a single 64-bit integer. The rightmost bit represents
+-- whether the user was active in the current day.
+    CAST(TRUE AS INT64) AS days_seen_bits,
+    CAST(seen_whats_new AS INT64) AS days_seen_whats_new_bits,
```

Comment:
I added this `CAST` here to encode the `seen_whats_new` boolean column into a bit integer, does it make sense?

Reply:
Yes, this looks perfect.